### DISCUSSION TITLE
[Question] flet app behaves differently on web vs flet window

### DISCUSSION THREAD

**Discussion by plugeit** [flet-dev/flet#1913] [Q&A] [answered]
import flet as ft

async def main(page: ft.Page):

    page.horizontal_alignment = ft.CrossAxisAlignment.CENTER
    page.padding=22
    page.scroll = "hidden"
    page.bgcolor="#F5F5F5"

    size_=30
    weight_=ft.FontWeight.W_900
    #---------------------------------------------navigation bar(footer)------------------------------------------------------------ 
    play_text=ft.Row(controls=[ft.Row(controls=[ft.Text("P", size=size_, weight=weight_, color="#F73F6F"),
                                        ft.Text("l", size=size_, weight=weight_, color="#F76F6F"),
                                        ft.Text("a", size=size_, weight=weight_, color="#FA755B"),
                                        ft.Text("y", size=size_, weight=weight_, color="#FEA743"),],spacing=1),
                                        ], alignment="center", spacing =10)
    page.navigation_bar = ft.NavigationBar(
        destinations=[
            ft.NavigationDestination(
                icon=ft.icons.HOME_OUTLINED,
                selected_icon=ft.icons.HOME_ROUNDED,
                label="Home",
                ),
            ft.NavigationDestination(icon_content=play_text,
                                     label="Book seat"),
            ft.NavigationDestination(
                icon=ft.icons.ACCOUNT_CIRCLE_OUTLINED,
                selected_icon=ft.icons.ACCOUNT_CIRCLE_ROUNDED,
                label="Account",
            ),
        ],
    )

    #---------------------------------------------------------end----------------------------------------------------------------------

    #---------------------------------------------Plug and Play Logo(header)------------------------------------------------------------ 
    
    size_=40
    plug_and_play_text=ft.Row(controls=[ft.Row(controls=[ft.Text("P", size=size_, weight=weight_, color="#F73F6F"),
                                        ft.Text("l", size=size_, weight=weight_, color="#F76F6F"),
                                        ft.Text("u", size=size_, weight=weight_, color="#FA755B"),
                                        ft.Text("g", size=size_, weight=weight_, color="#FEA743"),],spacing=1),

                                        ft.Row(controls=[ft.Text("&", size=size_, weight=weight_ , color="#FA755B")
                                        ]),

                                        ft.Row(controls=[ft.Text("P", size=size_, weight=weight_, color="#F73F6F"),
                                        ft.Text("l", size=size_, weight=weight_, color="#F76F6F"),
                                        ft.Text("a", size=size_, weight=weight_, color="#FA755B"),
                                        ft.Text("y", size=size_, weight=weight_, color="#FEA743"),],spacing=1),
                                        ], alignment="center", spacing =10)

    await page.add_async(plug_and_play_text)
    #---------------------------------------------------------end----------------------------------------------------------------------

    #---------------------------------------------body------------------------------------------------------------ 
    body_send_otp=ft.Column(controls=[ft.Text(),
                             ft.Container(content=ft.Column(controls=[ft.Row(controls=[ft.Text("Login / Sign Up", size=22, color="#062A70", text_align="center", weight=ft.FontWeight.W_900)], alignment="center"),
                                                                      ft.Row(controls=[ft.TextField(expand=True,height=53,suffix_icon=ft.icons.EMAIL_OUTLINED,label="Email")]),
                                                                      ft.Row(controls=[ft.TextField(expand=True,height=53,password=True, can_reveal_password=True,label="OTP")]),
                                                                      ft.Row(controls=[ft.Switch(label="  Remember me", value=True)]),
                                                                      ft.Row(controls=[ft.Container(ft.Column(controls=[ft.Row(controls=[ft.Text("Send OTP", text_align="center", weight=ft.FontWeight.W_900, color="#00A368")], alignment="center")], alignment="center"),border=ft.border.all(2, ft.colors.BLACK),border_radius=15, ink=True, bgcolor="#FFD630", expand=True, height=40)]),
                                                                      ], spacing=20),height=330, width=400, bgcolor="#D9D9D9", border_radius=15, padding=10)], alignment="center", spacing=50)
    #await page.add_async(body)
    body_sended_otp=ft.Column(controls=[ft.Text(),
                             ft.Container(content=ft.Column(controls=[ft.Row(controls=[ft.Text("Login / Sign Up", size=22, color="#062A70", text_align="center", weight=ft.FontWeight.W_900)], alignment="center"),
                                                                      ft.Row(controls=[ft.TextField(expand=True,height=53,suffix_icon=ft.icons.EMAIL_OUTLINED,label="Email")]),
                                                                      ft.Row(controls=[ft.TextField(expand=True,height=53,password=True, can_reveal_password=True,label="OTP")]),
                                                                      ft.Row(controls=[ft.Switch(label="  Remember me", value=True)]),
                                                                      ft.Row(controls=[ft.Container(ft.Column(controls=[ft.Row(controls=[ft.Text("Resend OTP", text_align="center", weight=ft.FontWeight.W_900, color="#00A368")], alignment="center")], alignment="center"),border=ft.border.all(2, ft.colors.BLACK),border_radius=15, ink=True, bgcolor="#FFD630", expand=True, height=40)]),
                                                                      ], spacing=20),height=330, width=400, bgcolor="#D9D9D9", border_radius=15, padding=10)], alignment="center", spacing=50)
    await page.add_async(body_send_otp)
    #----------------------------------------------end---------------------------------------------------------------------------------

ft.app(target=main, view=ft.WEB_BROWSER)

**Comment by ndonkoHenri** [comment]
Can you please explain the difference in behaviour?

**Reply by plugeit** [reply]
the text dosen't holds it's bold property when i switch to web view

**Comment by plugeit** [comment, marked as answer]
I think this is a bug i am going to raise an issue
